### PR TITLE
Add ETCD backup/restore docs for Bottlerocket

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
*Issue #, if available:*
https://github.com/aws/eks-anywhere/issues/5489

*Description of changes:*
Add ETCD backup/restore docs for Bottlerocket

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/tasks/cluster/etcd-backup-restore/bottlerocket-etcd-backup.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+You should now have the ETCD snapshot in your current working directory.
+
+### Restore ETCD from Backup
+Make sure to setup the [admin environment variables]({{< relref "#admin-machine-environment-variables-setup" >}}) and [prepare your ETCD nodes for restore]({{< relref "#prepare-etcd-nodes-for-backup-and-restore" >}}) before moving forward.
+
+1. Pause cluster reconciliation
+
+    Before starting the process of restoring ETCD, you have to pause some cluster reconciliation objects so EKS Anywhere doesn't try to perform any operations on the cluster while you restore the ETCD snapshot.
+    ```bash
+    # Pause control plane reconcilation
```
Comment (jiayiwang7) [inline docs/content/en/docs/tasks/cluster/etcd-backup-restore/bottlerocket-etcd-backup.md:102]: can we also include this to existing ubuntu etcd backup doc?

**Review by jiayiwang7 — COMMENTED** [review]
This is much better than the existing ubuntu etcd backup doc. Great job!
Mind do a sanity check on the existing ubuntu doc and see if there's anything we should update/add there?

File: docs/content/en/docs/tasks/cluster/etcd-backup-restore/bottlerocket-etcd-backup.md (hunk 1 above)
Comment (abhinavmpandey08) [reply docs/content/en/docs/tasks/cluster/etcd-backup-restore/bottlerocket-etcd-backup.md:102]: yep will add that

**Review by jiayiwang7 — APPROVED** [review]

**Pull request merged**